controllers/imageclusterinstall_controller.go

```diff
@@ -186,49 +202,72 @@ func (r *ClusterConfigReconciler) setHostConfiguredCondition(ctx context.Context
 		cond.Message = err.Error()
 	}
 
-	patch := client.MergeFrom(config.DeepCopy())
-	meta.SetStatusCondition(&config.Status.Conditions, cond)
-	return r.Status().Patch(ctx, config, patch)
+	patch := client.MergeFrom(clusterInstall.DeepCopy())
+	meta.SetStatusCondition(&clusterInstall.Status.ConfigConditions, cond)
```

Comment:
I see that nothing really changed, but can you please explain what's going on here?
What's the patch for?

Reply:
This is a more targeted way to update a resource. It is better at avoiding the resource version errors.

Reply:
Cool, thanks